workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectInitializationSystem.cs

```diff
@@ -12,38 +12,39 @@ namespace Improbable.Gdk.GameObjectCreation
     ///     and links it to the entity via the EntityGameObjectLinker. Also checks for entity removal and calls the
     ///     IEntityGameObjectCreator for cleanup.
     /// </summary>
-    [UpdateInGroup(typeof(SpatialOSReceiveGroup.EntityInitialisationGroup))]
     [DisableAutoCreation]
     internal class GameObjectInitializationSystem : ComponentSystem
     {
-        // TODO: Add our ISystemStateComponentData for all entities instead of ones with GameObjects, store spatial ID
-        // callbacks instead of in it, give that in removal callback instead of SpatialOSEntity struct.
+        public struct HasGameObjectSystemState : ISystemStateComponentData
+        {
+            public SpatialEntityId EntityId;
```

Comment:
Is it meant to contain a copy of another `IComponentData` here?

Reply:
should probably be directly the entity id. will change it